src/adapters/analytics/ga.js

```diff
@@ -40,53 +43,59 @@ exports.enableAnalytics = function ({ provider, options }) {
 
   var bid = null;
 
-  //first send all events fired before enableAnalytics called
+  if(_sampled) {
+    //first send all events fired before enableAnalytics called
 
-  var existingEvents = events.getEvents();
-  utils._each(existingEvents, function (eventObj) {
-    var args = eventObj.args;
-    if (!eventObj) {
-      return;
-    }
+    var existingEvents = events.getEvents();
 
-    if (eventObj.eventType === BID_REQUESTED) {
-      bid = args;
-      sendBidRequestToGa(bid);
-    } else if (eventObj.eventType === BID_RESPONSE) {
-      //bid is 2nd args
-      bid = args;
-      sendBidResponseToGa(bid);
+    utils._each(existingEvents, function (eventObj) {
+      var args = eventObj.args;
+      if (!eventObj) {
```

Comment:
it should be noted that these line changes were only from moving existing code around to be inside the `if(_sampled) {` block above.  However, this _is_ a bug; and even though it existed before this pull-request, now is as good as time as any to fix it.  Thanks!